ts/lib/auth/jwtaccess.ts

```diff
@@ -55,7 +57,8 @@ export default class JWTAccess {
    * @param {function} metadataCb a callback invoked with the jwt
    *                   request metadata.
    */
-  public getRequestMetadata(authURI: string, metadataCb) {
+  public getRequestMetadata(
+      authURI: string, metadataCb: (err: Error, headers: any) => void) {
```

Comment:
Made headers optional.  I'd like to take these `any`s and address them after we get the low hanging fruit here.